scripts/e2e-start-local-registry.js

```diff
@@ -1,8 +1,6 @@
-import { exec } from 'child_process';
+const { exec } = require('child_process');
 
-const verdacio = exec(
-  `yarn verdaccio --config ./scripts/local-registry/config.yml --listen 4872`
-);
+const verdacio = exec('npx verdaccio -c ./scripts/local-registry/config.yml');
```

Comment:
Why the switch from `yarn` to `npx`? We use yarn as the main package manager.

Reply:
https://github.com/nrwl/nx/blob/75cbd54818c78781c66cb5ae17ef2715a0d76680/scripts/local-registry.sh#L28-L32

Reply:
npx was used for the other verdaccio script

Reply:
That one should be changed to yarn as well. 😀

Reply:
@meeroslav Done